ebr_trackerbot/utility.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
I would name this file more specific like time_utility.py

Reply:
Done